app/services/metrc_service/plant/discard.rb

```diff
@@ -1,8 +1,9 @@
+require_relative 'discard_helpers'
```

Comment:
If this `discard_helpers` module is only used here, I think it's best to just define those constants directly in this file and keep everything in one place.

But that's a matter of preferences. I'll let you follow your heart.